ezyrb/ann.py

```diff
@@ -25,9 +25,47 @@ class ANN(Approximation):
     >>> print(y_pred)
     """
         
-    def __init__(self):
-        self.trained_epoch = 0   
-        self.criterion = torch.nn.MSELoss()
+    def __init__(self, layers, function, stop_training, loss=None):
```

Comment:
In the current implementation, it is not clear if the "function" input is really needed.